src/_gettsim/parameters/unterhalt.yaml

```diff
@@ -13,49 +13,130 @@ mindestunterhalt:
   unit: Euro
   reference_period: Month
   2016-01-01:
-    6: 335
-    12: 384
-    18: 450
+    1:
```

Comment:
I think a list would be better suited?

```yaml
- min_alter: 0
  max_alter: 5
  betrag: 335
- min_alter: 6
  max_alter: 11
  betrag: 384
- min_alter: 12
  max_alter: 17
  betrag: 450
```

Reply:
Oh I didn't even know I could do that. But how would the 'reference' key fit here? I
think we would want it to be there and not one level higher.

Reply:
Yeah, thinking of it, it probably won't work at this level. Sorry for the noise, trying to do too many things at once today...

Reply:
Still useful to know!